examples/face_recognition_11/README.md

```diff
@@ -0,0 +1,59 @@
+# Example Integration: Face Recognition (1:1)
+
+This example integration uses the [Labeled Faces in the Wild (LFW)](http://vis-www.cs.umass.edu/lfw/) dataset and Face Recognition (FR) workflow to demonstrate how to test and evaluate end-to-end FR (1:1) model pipelines on Kolena. The evaluation stages are: face detection (bbox), keypoint extraction, and recognition.
+
+## Setup
+
+This project uses [Poetry](https://python-poetry.org/) for packaging and Python dependency management. To get started,
+install project dependencies from [`pyproject.toml`](./pyproject.toml) by running:
+
+```shell
+poetry update && poetry install
+```
+
+## Usage
+
+The data for this example integration lives in the publicly accessible S3 bucket `s3://kolena-public-datasets`.
+
+First, ensure that the `KOLENA_TOKEN` environment variable is populated in your environment. See our
+[initialization documentation](https://docs.kolena.io/installing-kolena/#initialization) for details.
+
+This project defines two scripts that perform the following operations:
+
+1. [`seed_test_suite.py`](face_recognition_11/seed_test_suite.py) creates the following test suites:
+
+    - `labeled-faces-in-the-wild :: gender [FR]`, stratified by `gender` using image metadata
+    - `labeled-faces-in-the-wild :: race [FR]`, stratified by `race` using image metadata
+
+    Run this command to seed the default test suite:
+    ```shell
+    poetry run python3 face_recognition_11/seed_test_suite.py
+    ```
+
+    > **_NOTE:_**  Face bounding box and keypoint ground truths are not originally from LFW. They are collected from RetinaFace. Also, a sample size of 10K from LFW is used as the dataset instead.
```

Comment:
```suggestion
    > **NOTE:**  Face bounding box and keypoint ground truths are inferred from RetinaFace as they are not provided in LFW dataset. Also, for the demo purpose, we have subsampled 10,000 pairs, made up of 9,983 images, from LFW.
```

It would also be nice to add a link to RetinaFace model